SDL/include/WiringPiButtons.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef WIRINGPIBUTTONS_HPP
+#define WIRINGPIBUTTONS_HPP
+
+#include <stdio.h> 
+#include <wiringPi.h>
+#include <chrono>
+
+using namespace std::chrono;
+
+class WiringPiButtons
+{
+    public:
+      WiringPiButtons();
+      int getEvents();  
+
+    private:
+      enum Button { UP = 25, DOWN = 24, LEFT  = 23, RIGHT = 22, RELEASED = 0};
```

Comment:
Add a comment explaining the reasoning behind these numbers / the mapping to the physical pins.